### DISCUSSION TITLE
How to write background-position: -3px -5px, 0 0; in JIT mode

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#9586] [Help] [answered]
Hi, how write `background-position: -3px -5px, 0 0;` in JIT mode ? I found this `bg-[center_top_1rem]` but it now work when i write 
`bg-[-3px_-5px_0_0]`

**Comment by wongjn** [comment, marked as answer]
You can try giving a type-hint to Tailwind (since length values could be `background-position` or `background-size`): `bg-[position:-3px_-5px,0_0]`.

**Reply by IRediTOTO** [reply]
Thank you <3
